src/backend/distributed/executor/multi_utility.c

```diff
@@ -384,6 +375,11 @@ multi_ProcessUtility(Node *parsetree,
 		SetUserIdAndSecContext(savedUserId, savedSecurityContext);
 	}
 
+	if (ddlJob != NULL)
+	{
+		ExecuteDistributedDDLJob(ddlJob);
+	}
+
 	/* we run VacuumStmt after standard hook to benefit from its checks and locking */
 	if (IsA(parsetree, VacuumStmt))
```

Comment:
It easily could be. In fact, I could probably shoehorn `COPY` in here with enough effort. Didn't know how much of that you wanted.